### ISSUE TITLE
Implement Fisher's Exact Test for allele frequency differences

### ISSUE THREAD

**Issue by magicDGS** [magicDGS/popgenlib#6]

**Comment by magicDGS** [comment]
Because this is going to be required for a project depending on GATK4, maybe this can be implemented there directly using `org.broadinstitute.hellbender.utils.FisherExactTest.twoSidedPValue()`. Anyway, it would be nice to have our own implementation here for completeness...
